.travis.yml

```diff
@@ -14,6 +14,34 @@ matrix:
       script:
         - PYFLAKES_NODOCTEST=1 pyflakes scipy benchmarks/benchmarks | grep -E -v 'unable to detect undefined names|assigned to but never used|imported but unused|redefinition of unused' > test.out; cat test.out; test \! -s test.out
         - pep8 scipy benchmarks/benchmarks
+      after_success:
+        # run the doctests
+        - sudo apt-get update -qq
+        - sudo apt-get install -qq libatlas-dev libatlas-base-dev liblapack-dev gfortran
+        - sudo apt-get build-dep -qq python-matplotlib
+        - travis_retry pip install --install-option="--no-cython-compile" Cython==0.22
+        - travis_retry pip install $NUMPYSPEC
+        - travis_retry pip install nose argparse
+        - travis_retry pip install matplotlib
+        - python runtests.py --doctests-only -s linalg
```

Comment:
Is listing all the submodules really a good way to do this? The same problem turns up with refguide_check, where a testing script (or a user) needs to know a complete list of all the submodules in scipy to do proper testing. This seems prone to failure, in the form of somebody forgetting to mention a new submodule in one of the several places we'll have a list and the submodule failing to be tested. Surely there should be some central listing of submodules? Or, here, why not just omit the -s and do just one --doctests-only run?